llvm/lib/Transforms/Instrumentation/GCOVProfiling.cpp

```diff
@@ -914,6 +918,7 @@ bool GCOVProfiler::emitProfileNotes(
         GlobalVariable *Counters = new GlobalVariable(
             *M, CounterTy, false, GlobalValue::InternalLinkage,
             Constant::getNullValue(CounterTy), "__llvm_gcov_ctr");
+        Counters->setSection("__llvm_gcov_ctr_section");
```

Comment:
This is not coded as an AIX-only change. Please confirm if the code matches the intent.

Reply:
Good catch! Fixed!